### PR TITLE
Rescue connection errors when sending information to Zipkin fails

### PR DESCRIPTION
**Pull request by ykitamura-mdsol**
Rescue the following errors in `AsyncJsonApiClient#perform` method to make the error message more helpful:

- Net::ReadTimeout
- Faraday::ConnectionFailed

screen shot 2017-11-14 at 3 04 45 pm

@adriancole @jcarres-mdsol  @jfeltesse-mdsol

### CODE REVIEW

**Comment by jcarres-mdsol** [comment]
LGTM

File: lib/zipkin-tracer/zipkin_json_tracer.rb [hunk 1]
```diff
@@ -14,6 +14,10 @@ def perform(json_api_host, spans)
       req.headers['Content-Type'] = 'application/json'
       req.body = JSON.generate(spans_with_ips)
     end
+  rescue Net::ReadTimeout, Faraday::ConnectionFailed => e
+    error_message = "Got #{e.class.inspect} with Message #{e.message} error while connecting to #{json_api_host}. " \
+                    "- Please make sure the Zipkin server is running in that url and port."
```
Comment (jfeltesse-mdsol) [inline lib/zipkin-tracer/zipkin_json_tracer.rb]: I think "running in that URL and port" is not accurate
What about "... running at the specified location" or, .... nothing? I think "error while connecting to {host}" is explicit enough.

@ssteeg-mdsol ideas?

Comment (ssteeg-mdsol) [reply lib/zipkin-tracer/zipkin_json_tracer.rb]: How about
```
"Error while connecting to #{json_api_host}: #{e.class.inspect} with message '#{e.message}'. " \
Please make sure the URL / port are properly specified for the Zipkin server
```

**Comment by ykitamura-mdsol** [comment]
Updated the message:

screen shot 2017-11-14 at 3 53 19 pm

**Comment by ssteeg-mdsol** [comment]
Looks good to me 👍

**Pull request merged**